mlir/lib/Dialect/MIOpen/Transforms/LowerMIOpenOps.cpp

```diff
@@ -152,6 +152,7 @@ void LowerMIOpenOpsStep3Pass::runOnOperation() {
 void LowerMIOpenOpsStep4Pass::runOnOperation() {
   MLIRContext *ctx = &getContext();
   OwningRewritePatternList patterns(ctx);
+    patterns.insert<InWarpTransposeRewritePattern>(ctx);
```

Comment:
nit: clang-format would likely fail on this line?

Reply:
Fixed